0_intro/exercises/broken_ownership/src/lib.rs

```diff
@@ -1,27 +1,25 @@
 /// Create vector and fill it, then return it
 /// Help me `modify_values` won't give my vector back :(
-// FIXME
 pub fn get_answer() -> Vec<i32> {
     let mut values = vec![0, 0];
-    modify_values(values);
+    modify_values(&mut values);
     values
 }
 
-// FIXME
-fn modify_values(values: Vec<i32>) {
+fn modify_values(values: &mut Vec<i32>) {
     values[0] = 4;
     values[1] = 2;
 }
 
 /// Print the first word of the "hello world" message, then free the message
 /// Help me the borrow checker is bullying me :(
-// FIXME
 pub fn print_hello() {
     let mut msg = String::from("hello world");
+
     let word = first_word(&msg); // Get a reference to the first word
+    println!("The first word is: {}", word); // Print first word
```

Comment:
Very good! The order of statements in Rust matters a lot, as you can see. Always keep the borrow checker in mind and your code will become cleaner, shorter and overall better